src/mozanalysis/frequentist_stats/bootstrap.py

```diff
@@ -239,13 +237,13 @@ def compare_branches_quantiles(
         Ex: 0.2 is the 20th percentile, 0.5 is the median, etc.
     """
 
+    if quantiles_of_interest is None:
```

Comment:
Wow, I'm impressed at the scope of the change made by this tool. Effectively, this is changing the type of that parameter from `List[float]` to `List[float] | None`, is that ok?

Reply:
I think the rule that's triggering is that you shouldn't have mutable types as defaults, so we could always change the default to a tuple if you don't like this fix

Reply:
Definitely agree that mutable types as a default is a footgun. I suppose so long as this change passes all our tests, I'm good with it.